README.md

```diff
@@ -80,8 +80,8 @@ The defined milestones give detailed descriptions on what should be when achieve
 
 ##### Solid Pod
 
-- [Docker Let's Encrypt & NGiNX-proxy](https://github.com/nginx-proxy/docker-letsencrypt-nginx-proxy-companion)
-- [DigitalOcean NGiNX reverse proxy](https://www.digitalocean.com/community/tutorials/how-to-configure-nginx-as-a-web-server-and-reverse-proxy-for-apache-on-one-ubuntu-20-04-server)
+- [Docker Let's Encrypt & NGINX-proxy](https://github.com/nginx-proxy/docker-letsencrypt-nginx-proxy-companion)
+- [DigitalOcean NGINX reverse proxy](https://www.digitalocean.com/community/tutorials/how-to-configure-nginx-as-a-web-server-and-reverse-proxy-for-apache-on-one-ubuntu-20-04-server)
```

Comment:
I think nginx has gone through a couple of branding iterations. At this point, they seem to use either `nginx` or `NGINX` and not `NGiNX`.